### ISSUE TITLE
[Support] Cannot create cluster

### ISSUE THREAD

**Issue by jbmcfarlin31** [CrunchyData/postgres-operator#860]
**Which example are you working with?**
Creating a cluster following the docs [here](https://access.crunchydata.com/documentation/postgres-operator/4.0.0/operatorcli/pgo-overview/)

**What is the current behavior?**
When I run `pgo create cluster mycluster -n pgouser1` I get an `Unauthorized` error. I followed all of the instructions on the installation guide and everything is showing as successful and running.

**What is the expected behavior?**
The cluster should've been created

**Other information** (e.g. detailed explanation, related issues,  etc)
Log output from the pod apiserver:
```
time="2019-07-09T19:01:33Z" level=debug msg="clusterservice.CreateClusterHandler called"
time="2019-07-09T19:01:33Z" level=debug msg="Authentication Attempt CreateCluster username=[username]"
time="2019-07-09T19:01:33Z" level=debug msg="BasicAuthzCheck pgoadmin CreateCluster true"
time="2019-07-09T19:01:33Z" level=debug msg="Authentication Success"
time="2019-07-09T19:01:33Z" level=debug msg="GetNamespace username [username] ns [pgouser1]"
time="2019-07-09T19:01:33Z" level=debug msg="WatchingNamespace [pgouser1]"
time="2019-07-09T19:01:33Z" level=debug msg="create cluster called for mycluster2"
time="2019-07-09T19:01:33Z" level=error msg="error getting cluster Unauthorized"
time="2019-07-09T19:01:33Z" level=debug msg="pgcluster mycluster2 not found so we will create it"
time="2019-07-09T19:01:33Z" level=error msg=Unauthorized
time="2019-07-09T19:01:33Z" level=debug msg=userLabelsMap
time="2019-07-09T19:01:33Z" level=debug msg="map[crunchy_collect:false]"
time="2019-07-09T19:01:33Z" level=error msg=Unauthorized
time="2019-07-09T19:01:33Z" level=debug msg="Pgo.DefaultContainerResources is "
time="2019-07-09T19:01:33Z" level=debug msg="Pgo.Cluster.Policies "
time="2019-07-09T19:01:33Z" level=debug msg="[0] is Pgo.Cluster.Replicas"
time="2019-07-09T19:01:33Z" level=debug msg="replica count is 0"
time="2019-07-09T19:01:33Z" level=debug msg=5432
time="2019-07-09T19:01:33Z" level=debug msg="Pgo.Cluster.User is testuser"
time="2019-07-09T19:01:33Z" level=debug msg="Pgo.Cluster.Database is userdb"
time="2019-07-09T19:01:33Z" level=debug msg=1
time="2019-07-09T19:01:33Z" level=debug msg="backrest set to true in user labels"
time="2019-07-09T19:01:33Z" level=debug msg=" is Pgo.Cluster.Policies"
time="2019-07-09T19:01:33Z" level=debug msg="no policies are specified in either pgo.yaml or from user"
time="2019-07-09T19:01:33Z" level=error msg=Unauthorized
time="2019-07-09T19:01:33Z" level=error msg=Unauthorized
time="2019-07-09T19:01:33Z" level=error msg="error creating secret mycluster2-postgres-secret"
time="2019-07-09T19:01:33Z" level=debug msg="created secret mycluster2-postgres-secret"
time="2019-07-09T19:01:33Z" level=error msg="error creating secret mycluster2-postgres-secret Unauthorized"
```

**Please tell us about your environment:**

* Operating System: `Ubuntu 16.04`
* Where is this running ( Local , Cloud Provider): `Azure Stack`
* Storage being used (NFS, Hostpath, Gluster, etc): `StorageClass - specifically rook`
* Container Image Tag:
* PostgreSQL Version:
* Platform (Docker, Kubernetes, OpenShift): `Kubernetes via RKE v1.13.5`
* Platform Version: ```pgo client version 4.0.0
pgo-apiserver version 4.0.0
```

**Comment by jmccormick2001** [comment]
you might verify that 'installrbac' was run, that will set up the RBAC in the pgo/pgouser1/pgouser2 namespaces that would allow Secrets to be created.

**Comment by jbmcfarlin31** [comment]
@jmccormick2001 so the following outputs

kubectl get ns shows: `pgo, pgouser1, pgouser2`

kubectl get clusterrole shows: `pgopclusterrole, pgopclusterrolecrd, pgopclusterrolesecret`

kubectl get clusterrolebinding shows: `pgopclusterbinding-pgo, pgopclusterbindingcrd-pgo, pgopclusterbindingsecret-pgo`

Then when looking at the secrets for all `pgo, pgouser1, pgouser2` namespaces I see:
```
root@init-builder-01:~/pgo-operator# kubectl -n pgo get secret
NAME                            TYPE                                  DATA   AGE
default-token-7xhph             kubernetes.io/service-account-token   3      6d1h
pgo-backrest-repo-config        Opaque                                4      6d
pgo.tls                         kubernetes.io/tls                     2      6d
postgres-operator-token-jr7hm   kubernetes.io/service-account-token   3      5d22h
root@init-builder-01:~/pgo-operator# kubectl -n pgouser1 get secret
NAME                       TYPE                                  DATA   AGE
default-token-7wch6        kubernetes.io/service-account-token   3      6d1h
pgo-backrest-token-m4rz9   kubernetes.io/service-account-token   3      5d22h
root@init-builder-01:~/pgo-operator# kubectl -n pgouser2 get secret
NAME                       TYPE                                  DATA   AGE
default-token-lc6dp        kubernetes.io/service-account-token   3      6d1h
pgo-backrest-token-wgp26   kubernetes.io/service-account-token   3      5d22h
```
I did check my installation notes when I was doing this, and I did run `make installrbac`. I will re-run it though and see if that changes anything.

**Comment by jbmcfarlin31** [comment]
@jmccormick2001 I think I figured out my problem beforehand... when I re-ran the `make installrbac` during my initial installation I don't think I re-ran `make deployoperator`. 

So I ran `make installrbac` and let it delete and regenerate the key pairs, and I then ran `make deployoperator`. I then copied the new `server.crt` and `server.key` files and successfully ran `pgo version` followed by `pgo create cluster mycluster -n pgouser1` which resulted in:
```
created Pgcluster mycluster
workflow id 0ff81d60-8339-490a-b30a-32c88c3ee2a3
```

Thanks for pointing me in the right direction, I knew it must've been something simple that I had missed.
